cmd/ketch/app.go

```diff
@@ -26,3 +31,80 @@ func newAppCmd(cfg config, out io.Writer) *cobra.Command {
 	cmd.AddCommand(newAppExportCmd(cfg, out, appExport))
 	return cmd
 }
+
+func appState(pods []apiv1.Pod) (state string) {
```

Comment:
Let's use a more structured approach, 

```golang
type AppState struct {
    RunningPods int
    ErrorrPods int
    DeployingPods int
}
func appState(pods []apiv1.Pod) AppState {

}

func (s AppState) String() string {
   
}
```

Reply:
Didn't get what is the profit? More code, more complexity in code, same result. Why we need to complicate that?